apps/foundation-demo/src/app/cart/components/cart/cart.component.html

```diff
@@ -1,10 +1,12 @@
 <div class="cart">
-  <div class="cart__title" *ngIf="subtitle">
-    <span class="cart__subtitle">{{subtitle}}</span>
+  <div class="cart__header" *ngIf="title">
```

Comment:
Why can't we use transclusion here? I feel like `title` is an unnecessary `Input`.

Reply:
@lderrickable

Reply:
I think we can. I'll do a commit for it

Reply:
So I missed the ngIf before. I think that's needed for instances of cart without a header, which I think happens in checkout somewhere. Is there a way to do that with transclusion? I don't think there is. @damienwebdev

Reply:
Why do we have to transclude or put it as an input if the text for the cart title is always going to be `Your Cart`